projects/storefrontlib/src/lib/ui/layout/header/header.module.ts

```diff
@@ -11,11 +11,17 @@ import { TertiaryBarComponent } from './tertiary-bar/tertiary-bar.component';
 import { LoginModule } from '../../../user/components/login/login.module';
 import { MobileMenuComponent } from './mobile-menu/mobile-menu.component';
 import { PwaModule } from '../../../pwa/pwa.module';
+import {
+  LanguageSelectorModule,
+  CurrencySelectorModule
+} from '../../../site-context';
 
 @NgModule({
```

Comment:
I believe we should skip exporting everything in this module other than the HeaderComponent.